deltacat/compute/compactor_v2/constants.py

```diff
@@ -92,3 +92,16 @@
 SHA1_HASHING_FOR_MEMORY_OPTIMIZATION_DISABLED = env_bool(
     "SHA1_HASHING_FOR_MEMORY_OPTIMIZATION_DISABLED", False
 )
+
+# Whether to enable logging if any partition is found
+# to be non-compliant with the bucketing spec.
+ENABLE_BUCKETING_SPEC_COMPLIANCE_LOGGING = env_bool(
```

Comment:
What do you think about encapsulating this behavior within an `Enum`? Something like `BUCKETING_SPEC_COMPLIANCE_PROFILE`. It looks like logging and assertion are levels of severity on finding non-compliance with the bucketing spec? So logging would be on the side of a less profile that the user can select while assertion is more strict

Reply:
done.